plugins/catalog/src/components/AboutCard/AboutCard.tsx

```diff
@@ -120,6 +121,14 @@ export function AboutCard({ entity, variant }: AboutCardProps) {
                 entity.metadata.namespace || ENTITY_DEFAULT_NAMESPACE
               }/${entity.kind}/${entity.metadata.name}`}
             />
+            <IconLinkVertical
+              disabled={!entity.spec?.implementsApis}
+              label="View API"
+              icon={<BrightnessAutoIcon />}
+              href={`./${
```

Comment:
@freben does this look correct?

Reply:
I thought this would be just `./api` right?

Reply:
I thought the same. It was my first implementation but it was wrong.

Reply:
No I verified that this is actually correct, but I fail to understand why react-router thinks it's necessary. :( We could merge for now and then debug what's going on under the hood

Reply:
This also assumes that [this](https://github.com/spotify/backstage/blob/e11865ec69cb76bf03fcb415a51fc808c08b4337/packages/app/src/components/catalog/EntityPage.tsx#L179) route isn't modified in the app.

Reply:
fixed in #3037